src/lib.rs

```diff
@@ -27,16 +27,22 @@ extern crate log;
 extern crate serde;
 
 mod descriptor;
+#[cfg(feature = "electrum")]
+mod electrum;
+#[cfg(feature = "cli")]
```

Comment:
Runtime is used outside of cli, for instance in WASM. So we do not need this feature gate.

Reply:
Ok, 

I will need change `Runtime`, because `BlockchainExplorer` had **ElectrumClient** dependecy.

Reply:
I think we need to refactor `BlockchainExplorer` type into an enum, where ElectrumClient is just one of the options.

The whole point of this crate is to provide `Runtime`. The rest is done by `rgb-wallet` and `rgb-std`. Basically, `rgb` crate is just `rgb-wallet` crate + FS storage + electrum.